### PR TITLE
Footer documents

### PR DESCRIPTION
**Pull request by rekt-hard**

### CODE REVIEW

File: invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/overrides.less [hunk 1]
```diff
@@ -152,3 +152,8 @@ pre {
 
   }
 }
+
+a:hover .icon,
```
Comment (mb-wali) [inline invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/overrides.less:156]: could you double-check that its not overriding any other icon class.

**Review by mb-wali — COMMENTED** [review]
add a single screenshot.

File: invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/overrides.less (hunk 1 above)
Comment (rekt-hard) [reply invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/overrides.less:156]: should the red highlighting on hover only be in the footer element?

Comment (mb-wali) [reply invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/overrides.less:156]: yes.

**Comment by rekt-hard** [comment]
Only download icon (used for pdf documents) will be highlighted on hover

image

**Review by mb-wali — APPROVED** [review]

**Pull request merged**